README.md

```diff
@@ -40,10 +40,16 @@ Daedalus - Cryptocurrency Wallet
 
    substituters = https://cache.iog.io https://cache.nixos.org/
    trusted-substituters =
-   trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+   trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
```

Comment:
We still need this key, unfortunately it's named `hydra.iohk.io`, but really it is for `https://cache.iog.io` 😔

Reply:
I changed it back